[2025-05-17T17:00:13.584+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-17T17:00:13.612+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: retrain_PWU_dag.train_PWU manual__2025-05-17T17:00:12.702325+00:00 [queued]>
[2025-05-17T17:00:13.619+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: retrain_PWU_dag.train_PWU manual__2025-05-17T17:00:12.702325+00:00 [queued]>
[2025-05-17T17:00:13.620+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-05-17T17:00:13.632+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): train_PWU> on 2025-05-17 17:00:12.702325+00:00
[2025-05-17T17:00:13.640+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=19472) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-17T17:00:13.639+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'retrain_PWU_dag', 'train_PWU', 'manual__2025-05-17T17:00:12.702325+00:00', '--job-id', '393', '--raw', '--subdir', 'DAGS_FOLDER/retrain_PWU_dag.py', '--cfg-path', '/tmp/tmp9wpps8j2']
[2025-05-17T17:00:13.641+0000] {standard_task_runner.py:63} INFO - Started process 19473 to run task
[2025-05-17T17:00:13.642+0000] {standard_task_runner.py:91} INFO - Job 393: Subtask train_PWU
[2025-05-17T17:00:13.685+0000] {task_command.py:426} INFO - Running <TaskInstance: retrain_PWU_dag.train_PWU manual__2025-05-17T17:00:12.702325+00:00 [running]> on host b91ac73f2b68
[2025-05-17T17:00:13.770+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='retrain_PWU_dag' AIRFLOW_CTX_TASK_ID='train_PWU' AIRFLOW_CTX_EXECUTION_DATE='2025-05-17T17:00:12.702325+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-17T17:00:12.702325+00:00'
[2025-05-17T17:00:13.771+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-17T17:00:13.783+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-17T17:00:13.785+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/train_PWU.py']
[2025-05-17T17:00:13.792+0000] {subprocess.py:86} INFO - Output:
[2025-05-17T17:08:45.886+0000] {subprocess.py:93} INFO - [Epoch 1] Loss: 0.0091
[2025-05-17T17:08:45.887+0000] {subprocess.py:93} INFO - [✅ Saved] Trained model → /models/PWU/model.pth
[2025-05-17T17:08:46.455+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-17T17:08:46.457+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-17T17:08:46.481+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=retrain_PWU_dag, task_id=train_PWU, execution_date=20250517T170012, start_date=20250517T170013, end_date=20250517T170846
[2025-05-17T17:08:46.502+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-17T17:08:46.519+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-17T17:08:46.522+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
